.github/workflows/accessibility-testing.yml

```
@@ -0,0 +1,466 @@
+name: Healthcare Accessibility Testing
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)